gradle-baseline-java/src/main/groovy/com/palantir/baseline/plugins/BaselineCheckstyle.groovy

```diff
@@ -31,6 +31,7 @@ class BaselineCheckstyle extends AbstractBaselinePlugin {
     void apply(Project project) {
         this.project = project
 
+		project.rootProject.apply BaselineConfig
```

Comment:
I believe it's OK to apply plugins to other projects; you may need/want to check for duplicates though.